### ISSUE TITLE
include or download Info.plist 

### ISSUE THREAD

**Issue by carljmosca** [carljmosca/netbeans-macos-bundle#5]
It has been suggested that it would be easier if one only needed download the installation script.

**Comment by ulab** [comment]
This is fixed with the custom installer, isn't it?

**Comment by carljmosca** [comment]
Yes, that's correct but we have have some additional feedback on that script which we are looking into.
